contrib/perftest/README.md

```diff
@@ -0,0 +1,51 @@
+## perftest : tool for benchmarking and profiling libpod library
+perftest uses libpod as golang library and perform stress test and profile for CPU usage.
+
+Build:
+
+```
+# cd $GOPATH/src/github.com/containers/libpod/contrib/perftest
```

Comment:
An integration into the base Makefile would be nice, for example:
- `make perftest` for building
- `make run-perfest` for running